releases/7.6.0.md

```diff
@@ -185,6 +185,7 @@ Minor incompatibilities:
 -   `BaseImportModule.stage_files` is now abstract. (It used to call
     `stage_excel_files()`, which was only defined on certain subclasses.) `stage_files()`
     is now a more attractive target for overriding than `run_load_task()`.
+-   `GeojsonFeatureCollectionDataType` moved from `arches.app.datatypes.datatypes` to `arches.app.datatypes.core`.
```

Comment:
Unsure of the functional impact here if anyone was subclassing this -- the old import still works.